notesapi/v1/permissions.py

```diff
@@ -37,7 +37,7 @@ def has_permission(self, request, view):
             logger.debug("No token found in headers")
             return False
         try:
-            data = jwt.decode(token, settings.CLIENT_SECRET, audience=settings.CLIENT_ID)
+            data = jwt.decode(token, settings.CLIENT_SECRET, algorithms=["HS256"], audience=settings.CLIENT_ID)
```

Comment:
I thought we needed to pick up the algorithm from a setting? Also, why are IDAs decoding? I wonder what is special about this?

Reply:
Maybe we can try to trust the comment, assume this is fine, and test in Stage (or sandbox)?

Reply:
Picked up the `algorithm` from settings

Reply:
Yes. This was simply to bring this to your attention for some possible future, but not to make it an urgent blocking issue.

Reply:
@mikix: How do you feel about me leaving an in-context note on the topic via a comment? See https://github.com/openedx/edx-notes-api/pull/294